pkg/apis/local/v1alpha1/localvolumediscovery_types.go

```diff
@@ -0,0 +1,79 @@
+package v1alpha1
+
+import (
+	operatorv1 "github.com/openshift/api/operator/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// DiscoveryPhase defines the observed phase of the discovery process
+type DiscoveryPhase string
+
+// Different phases of the discovery process
+const (
+	// Discovering represents that the continuous discovery of devices is in progress
+	Discovering DiscoveryPhase = "Discovering"
+	// DiscoveryFailed represents that the discovery process has failed
+	DiscoveryFailed DiscoveryPhase = "DiscoveryFailed"
+)
+
+// LocalVolumeDiscoverySpec defines the desired state of LocalVolumeDiscovery
+type LocalVolumeDiscoverySpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
+	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
+
+	// Nodes on which the automatic detection policies must run.
+	// +optional
+	NodeSelector *corev1.NodeSelector `json:"nodeSelector,omitempty"`
+	// If specified tolerations is the list of toleration that is passed to the
+	// LocalVolumeDiscovery Daemon
+	// +optional
+	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
+}
+
+// LocalVolumeDiscoveryStatus defines the observed state of LocalVolumeDiscovery
+type LocalVolumeDiscoveryStatus struct {
+	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
```

Comment:
nit: I do not think these placeholder comments belong in final API.

Reply:
removed this comment along with a few other similar ones.